boards/neokey_trinkey/Cargo.toml

```diff
@@ -0,0 +1,74 @@
+[package]
+name = "neokey_trinkey"
+version = "0.1.0"
+authors = ["Broderick Carlin <broderick.carlin@gmail.com>"]
+description = "Board Support crate for the Adafruit Neokey Trinkey"
+keywords = ["no-std", "arm", "cortex-m", "embedded-hal"]
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/atsamd-rs/atsamd"
+readme = "README.md"
+edition = "2018"
+
+[dependencies]
+cortex-m = "0.6"
+embedded-hal = "0.2.3"
+nb = "0.1"
+
+[dependencies.cortex-m-rt]
+version = "0.6.12"
+optional = true
+
+[dependencies.atsamd-hal]
+path = "../../hal"
```

Comment:
This would make it a tier 1 board - not sure what the guidelines will be around new boards, if they should be tier 1 or tier 2 automatically

Reply:
The guidelines dictate that boards are Tier 2 by default. @BroderickCarlin, this means that the BSP should reference a crates.io version of `atsamd-hal` rather than a path dependency.